src/BarcodeScannerForPhoneGap/BarcodeScannerForPhoneGap.xml

```diff
@@ -31,5 +31,11 @@
             <description></description>
             <returnType type="Void"/>
         </property>
+        <property key="onchangeNanoFlow" type="nanoflow" required="false">
+            <caption>On change nanoflow</caption>
+            <category>Events</category>
+            <description></description>
```

Comment:
If there is no description then use `<description/>` instead

Reply:
Done